tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -115,3 +134,26 @@ def test_multiprocess(storage_url: str) -> None:
     assert len(trials) == n_workers * 20
 
     _check_trials(trials)
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_multiprocess_with_progbar(storage_url: str, n_jobs: int) -> None:
+    n_workers = 8
+    study_name = _STUDY_NAME
+    optuna.create_study(storage=storage_url, study_name=study_name)
+    with Pool(n_workers) as pool:
+        results = pool.map(
+            run_optimize_with_progbar, [(study_name, storage_url, n_jobs)] * n_workers
+        )
+        comp_cnt = 0
+        for result in results:
```

Comment:
Depending on the returned value by `pool.map`, but similarity `result` is not clear. One alternative might be as follows.

```suggestion
        for progress_bar in progress_bars:
```

Reply:
Thanks, all was fixed.